src/common/Tile/styled.js

```diff
@@ -1,128 +1,125 @@
-import styled from "styled-components";
+import styled from 'styled-components'
 
 export const StyledTile = styled.article`
-    background: ${({ theme }) => theme.color.white};
-    box-shadow: 0px 4px 12px 0px #BAC7D580; 
-    border-radius: 5px;
-    padding: 16px;
-    max-width: 288px;
-    display: flex;
+	background: ${({ theme }) => theme.color.white};
+	box-shadow: 0px 4px 12px 0px #bac7d580;
+	border-radius: 5px;
+	padding: 16px;
+	display: flex;
+
+	@media (min-width: 768px) {
+		min-width: 324px;
+		height: 650px;
+		display: flex;
+		flex-direction: column;
+		align-items: flex-start;
+	}
+`
 
-    @media (min-width: 768px) {
-        min-width: 324px;
-        display: flex;
-        flex-direction: column;
-        align-items: flex-start;
-    }
-`;
+export const Poster = styled.div`
+	background: ${({ theme }) => theme.color.silver};
+	width: 114px;
+	height: 169px;
+	border-radius: 5px;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+
+	@media (min-width: 768px) {
+		width: 292px;
+		min-height: 434px;
+	}
+`
 
-export const Image = styled.img`
-    width: 114px;
-    height: 169px;
-    border-radius: 5px;
+export const ImagePoster = styled.img`
+	width: ${({ $isCustom }) => ($isCustom ? '45px' : '114px')};
+	height: ${({ $isCustom }) => ($isCustom ? '45px' : '169px')};
+	border-radius: 5px;
 
-    @media (min-width: 768px) {
-        width: 292px;
-        height: 434px;
-    }
-`;
+	@media (min-width: 768px) {
+		width: ${({ $isCustom }) => ($isCustom ? '45px' : '292px')};
+		height: ${({ $isCustom }) => ($isCustom ? '45px' : '434px')};
+	}
+`
 
 export const TileContent = styled.div`
-    display: flex;
-    flex-direction: column;
-    margin-left: 16px;
-    padding-bottom: 4px;
-
-    @media (min-width: 768px) {
-        margin: 0;
-    }
+	display: grid;
+	padding-left: 16px;
+	height: fit-content;
+
+	@media (min-width: 768px) {
+		padding: 0;
+		height: 100%;
+	}
 `
-
-export const Title = styled.header`
-    font-size: 16px;
-    font-weight: 500;
-
-    @media (min-width: 768px) {
-        font-size: 22px;
-        padding: 16px 0px 8px;
-    }
+export const TileTitle = styled.header`
+	font-weight: 500;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	display: -webkit-box;
+	-webkit-line-clamp: 2;
+	-webkit-box-orient: vertical;
+
+	@media (min-width: 768px) {
+		font-size: 22px;
+		padding-top: 16px;
+        max-height: 65px;
```

Comment:
I suggest to change value of max-height to 67px bcs of some letters cut 
<img width="342" alt="image" src="https://github.com/pawelmroczek/movie-browser/assets/112400625/5058ada7-6539-41da-a7a7-e3181b1c8889">